src/api-calls.js

```diff
@@ -1,8 +1,4 @@
-import {
-  setDashboard
-} from './dom-updates'
-
-const getCustomersData = () => {
+const getAllCustomersData = () => {
```

Comment:
A small suggestion for drying this up, 
`getAllCustomersData`, `getBookingsData`, `getRoomsData`, and `getUserData` are all very similar. Is it possible to dry this up by using one function that takes an arg, and interpolating the arg for each URL?
Example arguments: 
- `customers`
- `bookings`
- `rooms`
- `customers/${id}`